File: CalculationTest/app/src/main/java/com/owulia/calculationtest/MainActivity.java
package com.owulia.calculationtest;

import androidx.appcompat.app.AppCompatActivity;
import androidx.navigation.NavController;
import androidx.navigation.Navigation;
import androidx.navigation.ui.NavigationUI;

import android.os.Bundle;

public class MainActivity extends AppCompatActivity {

    NavController controller;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        controller = Navigation.findNavController(this, R.id.fragment);
        NavigationUI.setupActionBarWithNavController(this, controller);

    }

    @Override
    public boolean onSupportNavigateUp() {

        if (controller.getCurrentDestination().getId() == R.id.questionFragment) {

        }

        return super.onSupportNavigateUp();
    }
}

File: WowCool/app/src/main/java/com/owulia/wowcool/MainActivity.kt
package com.owulia.wowcool

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.util.Log
import androidx.core.content.ContextCompat
import com.owulia.wowcool.fragment.HomeFragment
import com.owulia.wowcool.fragment.DemoFragment
import com.owulia.wowcool.fragment.MineFragment
import kotlinx.android.synthetic.main.activity_main.*

class MainActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
    }
}
